docs/source/quantization.rst

```diff
@@ -332,6 +335,95 @@ To learn more about quantization aware training, please see the `QAT
 tutorial
 <https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html>`_.
 
+FX Graph Mode Quantization
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Quantization types supported by FX Graph Mode can be classified in two ways:
+
+1.
+- Post Training Quantization (apply quantization after training, quantization parameters are calculated based on sample calibration data)
+- Quantization Aware Training (simulate quantization during training so that the quantization parameters can be learned together with the model using training data)
+
+2.
+- Weight Only Quantization (only weight is statically quantized)
+- Dynamic Quantization (weight is statically quantized, activation is dynamically quantized)
+- Static Quantization (both weight and activations are statically quantized)
+
+These two ways of classification are independent, so theoretically we can have 6 different types of quantization.
+
+The supported quantization types in FX Graph Mode Quantization are:
+- Post Training Quantization
+  - Weight Only Quantization
+  - Dynamic Quantization
+  - Static Quantization
+- Quantization Aware Training
+  - Static Quantization
+
+
+We have the following APIs in FX Graph Mode Quantization:
+- `prepare_fx`: prepares a model for post training quantization
+- `prepare_qat_fx`: prepares a model for quantization aware training
+- `convert_fx`: converts a prepared (and calibrated/trained) model to a quantized model
+- `fuse_fx`: This is not a function that automatically fuses module calls like Conv - BatchNorm into Conv. Currently only supports eval model.
+
+There are multiple quantization types in post training quantization (weight only, dynamic and static) and the configuration is done through `qconfig_dict` (an argument of the `prepare_fx` function), and mixed quantization type is also supported with the api (e.g. quantize the conv modules statically and quantize the linear module dynamically).
```

Comment:
optional: we could skip edge casey types until we have full docs (i.e we don't have to mention mixed quantization, etc)